xalan/src/test/java/org/apache/xalan/VersionTest.java

```diff
@@ -59,41 +85,47 @@ public void testCannotReadProperties() {
         .when(Version::getPropertiesStream)
         .thenThrow(NullPointerException.class);
       assertEquals("0.0.0", Version.readVersionNumber());
+      assertTrue(buffer.toString().contains("RuntimeException: Cannot read properties file"));
```

Comment:
`PrintStream` might buffer results so you probably want to add `mockPrintStream.flush()` before `buffer.toString()` (there are multiple uses of `buffer.toString()`, and it looks like all of them need adjustments